### PR TITLE
feat: roll up several quality of life updates

### PR DESCRIPTION
**Pull request by marcusrbrown**
The `textproto-grammar` extension hasn't seen updates in a while. This PR aims to bring the project up to date (and improve DX) with the following tasks:
- [x] Remove support for Node 12.
- [x] Update stale/vulnerable deps; deduplicate.
- [x] Add automatic dependency updates using [Renovate](https://github.com/renovatebot/renovate).
- [x] Use Prettier for consistent formatting (`{"tabWidth": 4, "trailingComma": "always"}`).

I opted for Renovate over GitHub's native Dependabot as I'm more familiar with it, and it can also handle security fixes in response to GitHub's dependency scans.

### CODE REVIEW

**Comment by marcusrbrown** [comment]
@thejustinwalsh This one should be good to go. The Renovate config is inactive because I wanted to get your input on noise level (what types of PRs do you want to be opened for approval vs. auto-merge). The action used in the `renovate` workflow requires a PAT with at least the `public:repo` scope, but action updates require particular scopes and possibly security considerations. I can raise those details in a separate issue.

Let me know if there's anything that needs to be modified in this PR. ✌🏽

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+fund=false
```
Comment (thejustinwalsh) [inline .npmrc:1]: 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,22 +24,22 @@ Ran npm version by mistake :rocket:
 
 ### 0.0.5 - 2020-12-08
 
-Epic PR from [@igetgames](https://github.com/thejustinwalsh/textproto-grammar/pull/5)!  
-Adds highlighting syntax within fenced code blocks in Markdown preview  
-Extension bundling with Rollup  
+Epic PR from [@marcusrbrown](https://github.com/thejustinwalsh/textproto-grammar/pull/5)!
```
Comment (thejustinwalsh) [inline CHANGELOG.md:27]: 🙌

File: README.md [hunk 3]
```diff
@@ -26,22 +26,22 @@ Ran npm version by mistake :rocket:
 
 ### 0.0.5 - 2020-12-08
 
-Epic PR from [@igetgames](https://github.com/thejustinwalsh/textproto-grammar/pull/5)!  
-Adds highlighting syntax within fenced code blocks in Markdown preview  
-Extension bundling with Rollup  
-CI workflow incuding release and publishing to VS Code Marketplace
+Epic PR from [@marcusrbrown](https://github.com/thejustinwalsh/textproto-grammar/pull/5)!
```
Comment (thejustinwalsh) [inline README.md:29]: 🙌

File: package.json [hunk 4]
```diff
@@ -81,35 +81,27 @@
...
-            "bundle.js"
-        ],
-        "rules": {
-            "indent": [
-                "error",
-                4
-            ],
-            "global-require": "off"
-        }
-    },
```
Comment (thejustinwalsh) [inline package.json:101]: 🚀

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -7,6 +7,9 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  NODE_VERSION: 16
+
```
Comment (thejustinwalsh) [inline .github/workflows/ci.yml:12]: 👌

**Review by thejustinwalsh — APPROVED** [review]

**Comment by thejustinwalsh** [comment]
> I can raise those details in a separate issue.

Thanks! Still very new to renovate, but down with the cause!

**Pull request merged**